angular/lib/scripts/verify-module.mjs

```diff
@@ -0,0 +1,48 @@
+import {glob} from 'glob';
+import path from 'path';
+import {fileURLToPath} from 'url';
+import {readFile, writeFile, mkdir, rm} from 'fs/promises';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const pathRegExp = /\\/g;
+export function normalizePath(str) {
+	return str.replace(pathRegExp, '/');
+}
+
+const files = (
+	await glob('**/*.component.ts', {
+		cwd: path.join(__dirname, '../src/components'),
```

Comment:
minor: i feel every time that the string is repeated 2+ times it is better to create a var for it (../src/components)